packages/stable/src/types.ts

```diff
@@ -1064,14 +1064,30 @@ export interface FileChange {
   };
 }
 
+export interface FileChangeByInstanceId {
```

Comment:
This contains duplicated field from https://github.com/cognitedata/cognite-sdk-js/pull/1166/files#diff-a1cb34e0ba200ad9a54dfae19192d251fe8573729eea371b18dacdbf2850ab44R1051-R1064
I suggest extracting the common fields out

Reply:
@BugGambit done